paris/src/main/java/com/airbnb/paris/ExtendableStyleBuilder.kt

```diff
@@ -0,0 +1,15 @@
+package com.airbnb.paris
+
+import android.view.View
+
+class ExtendableStyleBuilder<V : View> : StyleBuilder<ExtendableStyleBuilder<V>, StyleApplier<*, V>>() {
+
+    // Makes the builder public so that extensions can access it
+    public override var builder = super.builder
```

Comment:
does this need to be a var?

Reply:
It does. In `StyleBuilder` it can be "consumed" and then reset to a new instance. This happens when combining styles.

Reply:
I suppose I could add a `clear()` method to `ProgrammaticStyle.Builder` but that seems unusual for builders.